js/02/main.js

```diff
@@ -0,0 +1,31 @@
+//Use substr to slice out the phrase because because because from the following sentence:'You cannot end a sentence with because because because is a conjunction'
+let str = 'You cannot end a sentence with because because because is a conjunction';
+let substr = 'because because because';
+console.log(str);
+console.log(str.replace(substr));
```

Comment:
When you want to replace something you also need to provide a replacement. Currently pattern replaced with undefined.